reviews/review-03+2020-11-30.md

```diff
@@ -0,0 +1,169 @@
+software developers deliver reproducible science and build confidence that their
+work can be sustained even after their initial contributions.
+
+b. Notes/callouts and glossary are very helpful for the reader. Code
+instructions are clear, thorough, and nicely commented.
+
+2. What reservations do you have about this manuscript?
+
+a. Chapters on their own a solid. I think there are times where I felt myself
+wanting a bit more "glue" or context. For example, as currently presented,
+learning objectives are separated from chapters. I want to know at the beginning
+of a chapter: 1) How is this chapter going to make me a better RSE; 2) what are
+the specific objectives that are going to get me there. This content is all
+there, but why not make it more explicit? For example, in Chapter 4's intro you
+tell me that I may need to process thousands of files (good). But the first
+sentence comes out of nowhere (reused from some other text?) talking about GUI
+IDEs. A little bit of punchy rewriting can answer what I think is the primary
+question---How is this going to make be a better RSE.
+
+b. I wrote my answer to question 4 (comment a) before I finished the
+introduction to the first chapter. I feel like I am still not 100% convinced
+about the title. It works as is without being misleading, but I think revisions
+might focus this book a bit, and hopefully the authors will be open to change if
+they agree there are different ways to focus things.
+
+3. In terms of content and structure, do you think the chapters need any
+revisions, such as additions or omissions? Please be as specific as possible.
+
+a. Looking at the overall order of chapters---would it be beneficial to reorder
+in a way that more closely resembles how an RSE might create a project. This
+might suggest for example starting with Git to get things organized and under
+version control right away. Does chapter 7 on working with teams belong earlier
+as well? It felt orienting to see the chapter on automating analyses just after
+chapter on teamwork. I am thinking about the behaviors that go along with
+software development and placing emphasis on those in a clear context. I'm not
+convinced the current order is the best possible one. Perhaps storyboarding the
+entire book would help the authors discover opportunities to reorganize. The
+approach I suggest is a stylistic preference and might mean making chapters less
+monolithic, returning to topics as they are needed.
+
+b. Sec 1.1: Mentions "three" related concepts; why not give sustainability its
+own bullet point? I love your definition.
+
+c. Sec 1.1: Grammatical rewrite needed: "Nobody argues that research should be
+irreproducible or unsustainable, but "not against it" and actively supporting it
+are very different things. Academia doesn't yet know reward people for writing
+useful software, so while you may be thanked, the effort you put in may not
+translate into job security or decent pay."
```

Comment:
The grammatical re-write has been done during subsequent proof reading: "Academia doesn’t yet know *how to* reward people..."